target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8074-deco-x80-5g.dts

```diff
@@ -0,0 +1,469 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2022, Robert Marko <robimarko@gmail.com>
+altered by Jonathan Brophy <professor_jonny@hotmail.com>
+to suit the TP-Link Deco X80-5G target */
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-hk-cpu.dtsi"
+#include "ipq8074-ess.dtsi"
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	model = "TP-Link Deco X80-5G";
+	compatible = "tplink,deco-x80-5g", "qcom,ipq8074";
+
+	aliases {
+		 serial0 = &blsp1_uart5;
```

Comment:
No aliases for standard LED-s?

Reply:
The bootloader drives the led yellow on boot to indicate power until OpenWrt takes over, to mimic the factory led color I have driven the red and green RGB elements individually with the default-state = "on";
as I can't drive multiple LEDs from a single alias.

I have however done this with my virtual color driver as it is able to group the elements to virtual led's.
I will update this target in future once I have upstreamed my driver and added it to OpenWrt, unless you have another suggestion, as without userspace scripting it is impossible to implement the OEM colors that are assigned to the standard triggers under the QSDK.

https://github.com/professor-jonny/pj_openwrt/blob/kmod_virtual_led_test/target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8074-deco-x80-5g.dts#L21

Reply:
Ahh, I remeber it now.